<launch>
    <node pkg="assignment1" name="location_node" type="location.py" output="screen" />
</launch>
